### DISCUSSION TITLE
Latest version is still 5.5.3

### DISCUSSION THREAD

**Discussion by niklas2810** [firefly-iii/firefly-iii#4624] [Firefly III] [answered]
Hey! I just updated to 5.5.6 (via docker) and it seems like the latest version number hasn't been updated. When opening the dashboard, I get the following message: `Nachricht: Sie verwenden 5.5.6. Ihre Version ist neuer als die neueste Version (5.5.3). ` (Translation: `You're using 5.5.6. Your version is newer than the latest Version (5.5.3)`)

**Comment by JC5** [comment]
https://github.com/firefly-iii/firefly-iii/blob/5.5.6/config/firefly.php#L102

**Comment by JC5** [comment, marked as answer]
Oh wait that’s the version website. That updates every 24h.
